stats.go

```diff
@@ -43,10 +46,23 @@ func newStatsExporter(o Options) (*statsExporter, error) {
 		return nil, err
 	}
 
+	percentiles := make([]float64, len(o.HistogramPercentiles))
+	for n, percentileStr := range o.HistogramPercentiles {
+		percentile, err := strconv.ParseFloat(percentileStr, 64)
+		if err != nil {
+			return nil, fmt.Errorf("'HistogramPercentiles' must be in float format: Received %s", percentileStr)
+		}
+		if percentile < 0 || percentile > 1 {
+			return nil, fmt.Errorf("'HistogramPercentiles' must be between 0 and 1: Received %f", percentile)
```

Comment:
```suggestion
			return nil, fmt.Errorf("'HistogramPercentiles' must be between 0 and 1, got %f", percentile)
```

I am suggesting this because usually chained errors are separated by colons (`:`). If we start adding them into errors too, it will be just too many colons.

Reply:
Sure. Change applied.